src/Directory.Build.targets

```diff
@@ -5,7 +5,7 @@
   <!-- Used by Aspire.Workload.Testing.targets -->
   <Target Name="GetPackageName" Returns="@(PackageName)">
     <ItemGroup>
-      <PackageName Include="$(MSBuildProjectName)" Condition="'$(IsPackable)' == 'true'" />
+      <PackageName Include="$(MSBuildProjectName)" Condition="'$(IsPackable)' == 'true'" WithPackageVersion="$(PackageVersion)" />
```

Comment:
Just curious, why is this change needed?

Reply:
Because not all packages are going stable, so given this PR is stabilizing package versions, some will be like: `8.1.0` and some other swill be like `8.1.0-ci`. This change is ensuring that each project is returning the actual package version it will produce, as opposed to just assuming that all will have the same one.

Reply:
Ah, good idea!!